pkg/cloudprovider/providers/openstack/openstack_test.go

```diff
@@ -144,6 +144,98 @@ func TestToAuthOptions(t *testing.T) {
 	}
 }
 
+func TestCheckOpenStackOpts(t *testing.T) {
+	delay := new(MyDuration)
+	delay.Duration = 60 * time.Second
+	timeout := new(MyDuration)
+	timeout.Duration = 30 * time.Second
```

Comment:
Similar to earlier comment - these MyDuration objects should just be regular objects, not references.
```go
delay := MyDuration{60 * time.Second}
timeout := MyDuration{30 * time.Second}
```